### DISCUSSION TITLE
Chinese farming "pools"

### DISCUSSION THREAD

**Discussion by PPGOli** [Chia-Network/chia-blockchain#4184] [General]
Done

**Comment by cccat6** [comment]
The Chia official already warned to be careful of hpool.
Don't think it anymore. Just wait for the official pooling protocol out.

**Reply by PPGOli** [reply]
Thanks for your comment. Can you elaborate where to find this "warning"? There is not only Hpool, but also UUpool...

**Reply by cccat6** [reply]
Go for watch AMA last week.
